config/github-webhook/verify-signed-off.rb

```diff
@@ -0,0 +1,139 @@
+# to the ruby search path.  :-( There's probably a better way to do
+# this, but I'm not One with the Ruby...
+$LOAD_PATH.unshift '/usr/lib/ruby/gems/1.8/gems/crack-0.1.8/lib'
+$LOAD_PATH.unshift '/usr/lib/ruby/gems/1.8/gems/daemons-1.0.10/lib'
+$LOAD_PATH.unshift '/usr/lib/ruby/gems/1.8/gems/eventmachine-0.12.10/lib'
+$LOAD_PATH.unshift '/usr/lib/ruby/gems/1.8/gems/httparty-0.6.1/lib'
+$LOAD_PATH.unshift '/usr/lib/ruby/gems/1.8/gems/json-1.8.3/lib'
+$LOAD_PATH.unshift '/usr/lib/ruby/gems/1.8/gems/rack-1.1.0/lib'
+$LOAD_PATH.unshift '/usr/lib/ruby/gems/1.8/gems/rack-test-1.1.0/lib'
+$LOAD_PATH.unshift '/usr/lib/ruby/gems/1.8/gems/sinatra-1.0/lib'
+$LOAD_PATH.unshift '/usr/lib/ruby/gems/1.8/gems/thin-1.2.8/lib'
+
+require 'httparty'
+require 'sinatra'
+require 'json'
+
+# Put Sinatra on flatbed port 5000 (a fairly arbitrary choice)
+set :port, 5000
+
+# Globals
+user_agent = 'ofiwg/signed-off-by-checker'
+
+# Read the Github auth token in from the environment (it wouldn't do
+# the hard-code it where it would show up in a public Github repo!)
+auth_token = ENV['GITHUB_AUTH_TOKEN']
+if auth_token == nil then
+  puts "Someone forgot to set \$GITHUB_AUTH_TOKEN before launching me. Aborting!\n"
+  exit 1
+end
+
+# Github webhooks are not delivered as HTTP GETs.  If we don't have
+# this Sinatra method here, Sinatra displays a goofy error message if
+# someone just visits "https://url_to_this_script".
+get '/' do
+  'Nothing to see here but us chickens'
+end
+
+# Github delivers webhooks via HTTP POSTs.
+post '/' do
+  push = JSON.parse(request.body.read)
+  repo_name = push['repository']['full_name']
+
+  # If this is not a push on a pull request (i.e., if there's no
+  # commits to examine, such as if this is a test webhook ping from
+  # github), then just return HTTP status 200 (i.e., success) with a
+  # handy message that you can see in the Github webhook debug logs.
+  if push['action'] == nil || push['action'] != 'synchronize' then
+    return [200, 'This is not a pull request synchronize push; nothing for this bot to do!']
+  end
+
+  # This webhook will have only delivered the *new* commits on this
+  # PR.  We need to examine *all* the commits -- so we can discard the
+  # commits that were delivered in this webhook push.  Instead, do a
+  # fetch to get all the commits on this PR (note: putting the
+  # Authorization header in this request just in case this is a
+  # private repo).
+  commits_url = push['pull_request']['commits_url']
+  commits = HTTParty.get(commits_url,
+      :headers => {
+        'Content-Type'  => 'application/json',
+        'User-Agent'    => user_agent,
+        'Authorization' => "token #{auth_token}" }
+    )
+
+  # Sanity check: If we've got no commits, do nothing.
+  if commits.length == 0 then
+    return [200, 'Somehow there are no commits on this PR... Weird...']
+  end
+
+  # Ok, we have commits.  Go through them.
+  happy = true
+  targetURL = 'https://github.com/ofiwg/libfabric/wiki#how-to-contribute'
```

Comment:
I feel like we should link people [here](https://github.com/jsquyres/libfabric/blob/master/CONTRIBUTORS), whether directly or by adding a link to the wiki page.